packages/webapp/pages/notifications.tsx

```diff
@@ -105,16 +113,28 @@ const Notifications = (): ReactElement => {
         >
           {length > 0 &&
             queryResult.data.pages.map((page) =>
-              page.notifications.edges.map(
-                ({ node: { id, readAt, type, ...props } }) => (
-                  <NotificationItem
-                    key={id}
-                    {...props}
-                    type={type}
-                    isUnread={!readAt}
-                    onClick={() => onNotificationClick(id, type)}
-                  />
-                ),
+              page.notifications.edges.reduce(
+                (nodes, { node: { id, readAt, type, ...props } }) => {
+                  if (
+                    isSubscribed &&
+                    type === NotificationType.SquadSubscribeNotification
+                  ) {
+                    return nodes;
+                  }
+
+                  nodes.push(
+                    <NotificationItem
+                      key={id}
+                      {...props}
+                      type={type}
+                      isUnread={!readAt}
+                      onClick={(e) => onNotificationClick(e, id, type)}
```

Comment:
Why do we need the event now?

Reply:
Oh, this was just an unreverted change. Probably better to keep this as it is normal for event handlers to have this.